.github/workflows/publish.yml

```diff
@@ -1,27 +1,44 @@
-name: Publish
+name: "Lint and build code, and publish artifacts to repository"
+
 on:
-  create:
-    tags:
-      - '^[0-9]+\.[0-9]+\.[0-9]+$'
+  push:
+    branches:
+      - "main"
```

Comment:
How exactly does this work? What will the version of the published packages be? Wasn't the previous solution working (tags)?

Reply:
KmmBridge auto advances the patch based on the last tag, I'vr explained this in gradle.properties near the top.